Gruntfile.coffee

```diff
@@ -1329,6 +1345,15 @@ module.exports = (grunt) ->
 				options:
 					testname: "Local Test - <%= grunt.template.today('yyyy-mm-dd hh:MM') %>"
 
+		"string-replace":
+			inline:
+				files:
+					'dist/wet-boew/js/': 'dist/wet-boew/js/*.js'
+				options:
+					replacements: [
```

Comment:
Hmm good question. That would be important if the default parameters are changed over time as they have been. Good idea. I'll make that fix.